pkg/testcommon/testcommon.go

```diff
@@ -74,21 +77,21 @@ func (site *TestSite) Prepare() error {
 		return fmt.Errorf("Failed to CopyDir from %s to %s, err=%v", cachedSrcDir, site.Dir, err)
 	}
 
-	// If our test site has a Name: then update the config file to reflect that.
-	if site.Name != "" {
-		config, err := ddevapp.NewConfig(site.Dir, "")
-		if err != nil {
-			return errors.Errorf("Failed to read site config for site %s, dir %s, err:%v", site.Name, site.Dir, err)
-		}
-		err = config.Read()
-		if err != nil {
-			return errors.Errorf("Failed to read site config for site %s, dir %s, err: %v", site.Name, site.Dir, err)
-		}
-		config.Name = site.Name
-		err = config.Write()
-		if err != nil {
-			return errors.Errorf("Failed to write site config for site %s, dir %s, err: %v", site.Name, site.Dir, err)
-		}
+	// Create a config object. Err is ignored as we may not have
```

Comment:
Probably just say in the PR the reason for this set up changes.

Reply:
These changes were necessary to facilitate using a test site archive that does not contain a .ddev/config.yaml file, which is the case now that we're using the official d8 release. I also simplified a bit in a couple ways, by always setting to a test site name we define, and by removing the extra config.Read() (this is already called inside ddevapp.NewConfig()

Reply:
I'm actually pretty happy to not assume the pre-existing .ddev. Never did like that.